libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailManagementDialog/ScheduledEmails.tsx

```diff
@@ -5,19 +5,31 @@ import { FormattedMessage } from "react-intl";
 import { IScheduledMail } from "@gooddata/sdk-backend-spi";
 import { LoadingSpinner } from "@gooddata/sdk-ui-kit";
 import { ScheduledEmail } from "./ScheduledEmail";
+import { ITheme } from "@gooddata/sdk-backend-spi";
+import { withTheme } from "@gooddata/sdk-ui-theme-provider";
 
 interface IScheduledEmailsProps {
     onDelete: (scheduledEmail: IScheduledMail) => void;
     isLoading: boolean;
     scheduledEmails: IScheduledMail[];
     currentUserEmail?: string;
+    theme?: ITheme;
 }
 
-export const ScheduledEmails: React.FC<IScheduledEmailsProps> = (props) => {
-    const { isLoading, scheduledEmails, currentUserEmail, onDelete } = props;
+const RenderScheduledEmails: React.FC<IScheduledEmailsProps> = (props) => {
```

Comment:
I would rename it to something like `ScheduledEmailsComponent` when some HOC wrapping is used.